### ISSUE TITLE
Embedded field corrupted

### ISSUE THREAD

**Issue by sebastiandev** [orientechnologies/orientdb-studio#339]
When adding an embedded field, like a dictionary, when clicking save it turns the dict into a list and removes all keys, leaving only the values:

When trying to edit the *display configuration* I add an EMBEDDED field with content:

```javascript
{
  "width":2068.7778,
  "height":500,
  "classes": {
    "Person": {
      "fill":"#d62728",
      "stroke":"#951b1c",
      "icon":null,
       "display":"name"
     },
     "Knows": {
       "display":"@class"
      }
   },
   "node":{
      "r":30
    },
   "linkDistance":140,
   "charge":-1000,
   "friction":0.9,
   "gravity":0.1
}
```

Becomes:
```javascript
[
  2068.7778,
  500,
  {
    "Person": {
      "fill":"#d62728",
      "stroke":"#951b1c",
      "icon":null,
       "display":"name"
     },
     "Knows": {
       "display":"@class"
      }
  },
  {
      "r":30
  },
  140,
  -1000,
   0.9,
   0.1
]
```

**Studio version:** 2.1
**OrientDB version:** 2.1.5

**Comment by maggiolo00** [comment]
Hi @sebastiandev 
How di you add the embedded field?

**Comment by sebastiandev** [comment]
Through the UI. "Add field" selected embedded and also embeddedmap. Then
pasted the dict un the textedit.

When clicking save, the bad things happen

**Comment by maggiolo00** [comment]
Fixed in 2.1.x and develop
